paper-fab-speed-dial-action.html

```diff
@@ -56,7 +58,7 @@
 		</style>
 
 		<div class="flex"><span class="label"><slot></slot></span></div>
-		<paper-fab class="fab" icon=[[icon]] mini></paper-fab>
+		<paper-fab class="fab" icon$="[[icon]]" src$="[[src]]" mini></paper-fab>
```

Comment:
Why do you reflect the properties to the DOM (`$=`)? Better avoid this if there is no strong reason because reflecting to attributes is rather expensive.